components/local-app/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 A curated list of notable changes to the Gitpod CLI.
 
+## v1.0.4
+
+-  When opening / sshing into a stopped workspace, it will now be started automatically
+-  `--open`ing a workspace now writes the opened URL to the output
```

Comment:
typeos?

Reply:
it was intended, but phrased poorly. Thanks!